src/pysdk/grvt_signing.py

```diff
@@ -1,57 +1,19 @@
-from dataclasses import dataclass
 from enum import Enum
 
 from eth_account import Account
 from eth_account.messages import encode_typed_data
 
 from .grvt_api_base import GrvtApiConfig, GrvtEnv
-from .types import Currency, Kind, Order, TimeInForce
+from .types import Instrument, Order, TimeInForce
 
 #########################
 # INSTRUMENT CONVERSION #
 #########################
 
 
-class SignKind(Enum):
-    PERPETUAL = 1
-    FUTURE = 2
-    CALL = 3
-    PUT = 4
-
-
-KIND_TO_SIGN_KIND = {
-    Kind.PERPETUAL: SignKind.PERPETUAL,
-    Kind.FUTURE: SignKind.FUTURE,
-    Kind.CALL: SignKind.CALL,
-    Kind.PUT: SignKind.PUT,
-}
-
-
-class SignCurrency(Enum):
-    USDC = 2
-    USDT = 3
-    ETH = 4
-    BTC = 5
-
-
-CURRENCY_TO_SIGN_CURRENCY = {
-    Currency.USDC: SignCurrency.USDC,
-    Currency.USDT: SignCurrency.USDT,
-    Currency.ETH: SignCurrency.ETH,
-    Currency.BTC: SignCurrency.BTC,
-}
-
 PRICE_MULTIPLIER = 1_000_000_000
 
 
-SIGN_CURRENCY_TO_MULTIPLIER = {
-    SignCurrency.USDC: 1_000_000,
-    SignCurrency.USDT: 1_000_000,
-    SignCurrency.ETH: 1_000_000_000,
-    SignCurrency.BTC: 1_000_000_000,
-}
-
-
 class SignTimeInForce(Enum):
```

Comment:
This is still around. But shouldn't be a problem given there wont be new time in force.